ome/bioformats
# Update roi handler

To test : 

1) Add bioformats_package.jar to your ImageJ plugins folder
2) Open any Image in ImageJ, draw roi's and add them to the ROI Manager.
3) Try exporting images using Plugins-->Bio-Formats-->Bio-Formats Exporter. 

This should write the ROI's from the ROI manager to your ome-tiff.

To check : 

Using Command line tools : 

---. /showinf -nopix -omexml /Users/bramalingam/Desktop/ImageJTest/finalTest.ome.tif

This should show all the ROI's , within "< ROI >" " < /ROI >"

Using ImageJ (Post Export):
_Note : Please replace the develop version of Bioformats_package.jar in your ImageJ folder to 5.0._ version of Bioformats_package.jar, the current Develop version has a known issue while reading xml files with unicode unit and will not show the ROI's that were written earlier.

Import the Image using Bio-formats Importer (select checkbox "Display ROI's" while import) and check if the ROI Manager is populated with the ROI's (that were drawn earlier).

Also : 
4) Try overwriting an existing file, while exporting.
